src/components/Photo/DocumentLiveCapture.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import * as React from 'react'
+import { h, Component } from 'preact'
+import { screenshot } from '~utils/camera.js'
+import { mimeType } from '~utils/blob.js'
+import { sendEvent } from '../../Tracker'
+import { DocumentOverlay } from '../Overlay'
+import { ToggleFullScreen } from '../FullScreen'
+import Timeout from '../Timeout'
+import Camera from '../Camera'
+import CameraError from '../CameraError'
+import style from './style.css'
+
+type State = {
+  hasBecomeInactive: boolean,
+  hasCameraError: boolean,
+  snapshotBuffer: Array<{
+    blob: Blob
+  }>,
+}
+
+type Props = {
+  translate: (string, ?{}) => string,
+  onCapture: Function,
+  renderFallback: Function,
+  isUploadFallbackDisabled: boolean,
+  trackScreen: Function,
+  documentType: string,
+  className: string,
+  containerClassName: string,
+  renderTitle: Function,
+  renderError: Function
+}
+
+const screenshotQuality = 0.95
+
+export default class DocumentLiveCapture extends Component<Props, State> {
+  webcam = null
+  snapshotIntervalRef: ?IntervalID = null
+  screenshotQuality = 0.95
```

Comment:
`screenshotQuality` is defined twice. Is `const screenshotQuality = 0.95` necessary?

Reply:
My bad. Missed that when I was cleaning things up.